scripts/run-test262.ts

```diff
@@ -1,8 +1,11 @@
 import { parseScript, parseModule } from '../src/meriyah';
 import * as path from 'path';
 import * as fs from 'fs';
+import * as semver from 'semver';
 import run = require('test262-parser-runner');
 
+const vFlagSupported = semver.gte(process.version.slice(1), '20.0.0');
```

Comment:
Why don't we use `new RegExp('', 'v')` to test?

Reply:
```suggestion
const vFlagSupported = (() => {
  try {
    return new RegExp('', 'v').flags === ‘v’
  } catch {}
})();
```

Reply:
Awesome!